lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -158,10 +158,18 @@ def get(self):
                     },
                 }
                 url = self.param_template.sub("", url)
-                path = url[len(self.uri_prefix) + 1 :]
-                path = path.replace("/", "_")
```

Comment:
I think I kind of liked the old approach here (er, I think I like it better than having to explicitly provide what seems like redundant information).  We can't just extend the previous heuristic to chop off the tail at the first occurrence of a parameter (like we're chopping off the head after the URI prefix)?

Reply:
I'm going to move this comment to #2956.